source/how-tos/debug/debug-apache.rst

```diff
@@ -77,7 +79,39 @@ Or you're using the wrong hostname in your browser.
 
           sudo /sbin/apache2 -S
 
+Performance Tuning
+------------------
+
+If you're servicing many clients at time (more than 50) you will likely need to change the
+`Apache Httpd's worker configuration`_. The default configuration may degrade service when
+Httpd has to serve many clients (I.e., when you have a lot of customers using Open OnDemand).
+
+We suggest configurations similar to this. 
+
+.. warning:: 
+  `ServerLimit` and `ThreadsPerChild` should not exceed the number of cores available. 8
+  is given in the example, but will likely need to change.
+
+.. code-block:: apache
+
+  # /opt/rh/httpd24/root/etc/httpd/conf.d/mpm.conf
```

Comment:
This is only the path on RHEL7 and this path is specific to Puppet.  The path on default install is `/opt/rh/httpd24/root/etc/httpd/conf.modules.d/00-mpm.conf`

Reply:
Also don't put module configs in `conf.d` that's for VirtualHost configs, module configs in modern Apache got in `conf.modules.d` at least on RHEL hosts.